spec/factories/users.rb

```diff
@@ -192,6 +192,12 @@
       end
     end
 
+    trait :with_pending_in_person_enrollment do
+      after :build do |user|
+        create(:in_person_enrollment, :pending, user: user)
```

Comment:
should this have a profile too?

Reply:
Added in 945454348626ae9a2c46d75db05f99301b917db3